README.md

```diff
@@ -1,20 +1,68 @@
-# eslint-config-10up
+# 10up Eslint Config
 
 > ESLint [shareable config](https://github.com/10up/eslint-config)
 
 [![Support Level](https://img.shields.io/badge/support-active-green.svg)](#support-level)
 
-[eslint-config-10up](https://github.com/10up/eslint-config) is a shareable configuration package for [eslint](http://eslint.org).
+[@10up/eslint-config](https://github.com/10up/eslint-config) is a shareable configuration package for [eslint](http://eslint.org) built on top of [eslint-airbnb-config](https://github.com/airbnb/javascript) and modified to meet 10up's own standards.
 
 ## Installation
 
-To use this config, install [@10up/eslint-config](https://github.com/10up/eslint-config) as a development dependecy of your project:
+1. Install all peer dependencies
+
+```sh
+npx install-peerdeps --dev @10up/eslint-config@next
+```
+
+2. Install [@10up/eslint-config](https://github.com/10up/eslint-config) as a development dependecy of your project:
 
 ```sh
 npm install @10up/eslint-config --save-dev
 ```
 
-Next, add this configuration to your `package.json`:
+## Avaliable ESLint configs
+
+### @10up/eslint-config
+
+The default export and contains common rules that are not specific to any framework or environment.
+
+### @10up/eslint-config/react
+
+Extends `@10up/eslint-config` adding specific rules to react.
+
+This rule requires installing `babel-eslint`
+
+```sh
+npm install --save-dev 'babel-eslint
```

Comment:
Remove single quote before babel-eslint. It should be `npm install --save-dev babel-eslint`

This is breaking rest of the page.